docs_user/modules/proc_configuring-networking-for-control-plane-services.adoc

```diff
@@ -15,6 +15,9 @@ For example,
 ```
 apiVersion: k8s.cni.cncf.io/v1
 kind: NetworkAttachmentDefinition
+metadata:
+  name: internalapi
+  namespace: openstack
 spec:
```

Comment:
@archana203 This doc update is from a bug you submitted: https://issues.redhat.com/browse/OSPRH-8468

Reply:
Lgtm